### ISSUE TITLE
syntax: ;-separated commands get column-aligned when using -kp

### ISSUE THREAD

**Issue by Carpetsmoker** [mvdan/sh#285]
```
$ cat test.sh
case "h" in
        h) _usage;    exit 0    ;;
esac

ls /; ls /etc

$ shfmt -kp test.sh
case "h" in
 h)
           _usage
                      exit 0
                                ;;
esac

ls /
      ls /etc
```

As you can see, the `;`-separated commands get aligned in "columns" on
different lines, which strikes me as a bug. I would either expect it to be
expanded to several lines:

```
ls /
ls /etc
```

Or be preserved as columns (with spacing, if any):

```
case "h" in
        h) _usage;    exit 0    ;;
esac
```

**Comment by mvdan** [comment]
This is likely because the formatter currently doesn't allow multiple commands to be chained in a single line (nesting is fine, though). See #261.

Once that issue is fixed, this should be fixed too. It's hard to predict whether the two options will play well together though, so I'll keep this issue open until we can confirm either way.

**Comment by mvdan** [comment]
@Carpetsmoker what shfmt version is that? on master I get:

```
$ cat f.sh

case "h" in
        h) _usage;    exit 0    ;;
esac

ls /; ls /etc
$ shfmt -kp f.sh
case "h" in
h)
        _usage
        exit                      0
        ;;
esac

ls /
ls       /etc
```

Perhaps not ideal, but certainly not as broken as the output you showed.

**Comment by Carpetsmoker** [comment]
It was master at the time of the report; guess things changed a bit since then :-)

I get the same results as you when I try again with master.

**Comment by mvdan** [comment]
Actually, that is just because of the fix for #286.

I don't think there's anything else to fix here besides #261, unless I'm missing something?

**Comment by mvdan** [comment]
I'll assume there's nothing else to fix here, besides the other open issue :)
